tools/istio-iptables/pkg/validation/linux_test.go

```diff
@@ -17,14 +17,27 @@ package validation
 import (
 	"encoding/binary"
 	"testing"
-
-	"github.com/coreos/etcd/pkg/cpuutil"
+	"unsafe"
 )
 
+const intWidth int = int(unsafe.Sizeof(0))
+
+var byteOrder binary.ByteOrder
+
+// Inspired by etcd cpuutil
+func init() {
```

Comment:
Remove this. It's a duplicate of what's done in `linux.go` in the same directory.

Reply:
thanks, removed this